src/CommandOption/AllCommandsOption.php

```diff
@@ -2,7 +2,7 @@
 
 declare(strict_types=1);
 
-namespace Huttopia\ConsoleBundle;
+namespace Huttopia\ConsoleBundle\CommandOption;
```

Comment:
If you have a better directory for `AllCommandsOption`, i don't want to move it into `Command` directory because it's not a Command.